### ISSUE TITLE
Opinions on simplifying the text loading animation?

### ISSUE THREAD

**Issue by marcustyphoon** [ManuelHaag/gw2-ui#9]
I was working on improving some performance things and noticed that the `Spinner` animation used to replace icons in the loading state is much more performance efficient than the `Progress` animation used for text (CSS is weird, don't ask). While trying to improve this, I got to thinking... is there any point of having both animations in the first place? ~~Right now they're only used together, so only one needs an animation to indicate the loading state, and having both feels pretty visually noisy to me. Thus, I propose making one static, probably `Progress`.~~

**Comment by gw2princeps** [comment]
I agree. We could replace the text progress with a skeleton component https://material-ui.com/components/skeleton/
Not sure how performant this is, but there must be a reason why this one is so frequently used. 
Thoughts?

**Comment by marcustyphoon** [comment]
Ah, very nice. The `wave` version of the MUI skeleton component is implemented in the way I was doing it/the way I did the performance improvement to the traitline connectors, using a css `translate` on a subelement within an overflow-hidden container. (Translate effects can be run off the main thread; background-position cannot.)

Performance aside, I also like this version aesthetically. It's subtle but clear.

**Comment by gw2princeps** [comment]
Alright, then this is a done deal.

**Comment by marcustyphoon** [comment]
I had forgotten about the ability to disable the icons, so ignore what I said about removing the animation completely!
